frontend/src/bundles/studio/components/timeline/rows/scenes-row.tsx

```diff
@@ -48,6 +96,13 @@ const ScenesRow: React.FC = () => {
                             height="100%"
                         />
                     )}
+                    <CloseButton
+                        onClick={handleDeleteClick}
+                        color="gray.300"
+                        data-id={item.id}
+                        variant={'simple'}
```

Comment:
```suggestion
                        variant='simple'
```
And the same for other components